keps/sig-apps/592-ttl-after-finish/README.md

```diff
@@ -250,17 +257,141 @@ Mitigations:
 
 ## Graduation Criteria
 
-We want to implement this feature for Pods/Jobs first to gather feedback, and
-decide whether to generalize it to custom resources. This feature can be
-promoted to beta after we finalize the decision for whether to generalize it or
-not, and when it satisfies users' need for cleaning up finished resource
-objects, without regressions.
+- The feature implemented for Job, as future work, it can be extended to Pods and perhaps custom resources, but that should happen under separate feature flags.
+- Add necessary tests
+- Graduate to Beta in v1.21
```

Comment:
Graduation criteria should be based on quality/maturity instead of releases. This can be captured in the `milestone` of kep.yaml.

Reply:
Agree with Janet. Also please split by alpha-beta and beta->ga graduation, see e.g. here:
https://github.com/kubernetes/enhancements/tree/master/keps/sig-api-machinery/1904-efficient-watch-resumption#graduation-criteria

Reply:
Updated.